runners/direct-java/src/main/java/org/apache/beam/runners/direct/BoundedReadEvaluatorFactory.java

```diff
@@ -73,7 +73,7 @@ public void cleanup() {}
    * already done so.
    */
   private <OutputT> TransformEvaluator<?> getTransformEvaluator(
-      final AppliedPTransform<?, PCollection<OutputT>, Bounded<OutputT>> transform) {
```

Comment:
Can you add this to the `.travis.yaml` config? I'm worried that without automated validation, Eclipse will quickly get broken again by devs using other IDE's.